README.md

```diff
@@ -38,7 +38,19 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
 The passcode will expire after one week. To generate a new passcode:
 
 1. Run `twilio rtc:apps:video:delete`.
-1. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+2. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+
+#### Troubleshooting The Twilio CLI
+
+If any errors occur after running the `twilio rtc:apps:video:deploy --authentication passcode` command, then run the following commands:
```

Comment:
Should we be less specific and just say hey if you run into (basically any) CLI problem you can try these commands? Mainly thinking we could avoid the duplicate maintenance of this command syntax. If syntax changes not sure we will remember to change here. Not a big deal though.